screens/RegisterScreen.tsx

```diff
@@ -62,16 +63,15 @@ export default function RegisterScreen({ navigation }) {
         <Text style={ButtonStyles.buttonText}>Register</Text>
       </TouchableOpacity>
 
-      <View style={{marginTop: 32, alignItems: 'center'}}>
+      <View style={{ marginTop: 32, alignItems: "center" }}>
```

Comment:
Hm, I need to look at my linter. thx for setting to double quotes

Reply:
No worries this came from my auto format on save lol